packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1001,10 +1012,25 @@ export default class Meeting extends StatelessWebexPlugin {
         }
 
         this.requiredCaptcha = err.captchaInfo;
+
+        Metrics.sendBehavioralMetric(
+          BEHAVIORAL_METRICS.VERIFY_CAPTCHA_ERROR,
+          {
+            reason: err.message,
+            stack: err.stack
+          }
+        );
         throw (new CaptchaError());
       }
       else {
         this.meetingInfoFailureReason = MEETING_INFO_FAILURE_REASON.OTHER;
+        Metrics.sendBehavioralMetric(
```

Comment:
Change this to fetch meeting info failure  , This is the global error catch 

Add the metrics for adhoc failure here

Reply:
Done..